features/db.feature

```diff
@@ -21,6 +21,9 @@ Feature: Perform database operations
     When I try the previous command again
     Then the return code should be 1
 
+    When I run `wp db check`
+    Then STDOUT should not be empty
```

Comment:
Ok. That's off-topic for the pull request, and not really appropriate for the project. I'm going to delete the comment. Please keep conversations on-topic.